store-and-retrieve-data/use-the-wayne-dapp.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Use the Wayne dApp
+description: Use the Wayne web dApp to create buckets, upload files, and download data on the DataHaven testnet without writing code.
+---
+
+# Use the Wayne dApp
+
+The Wayne dApp is a convenient front-end for users to interact with the DataHaven network. It guides you through connecting your wallet, creating buckets, and uploading or downloading files without writing any code. **Buckets created in Wayne are currently public, meaning anyone can access the data stored in them.** There are no access controls or privacy features at this time; private buckets will be added in a future release.
+
+## Prerequisites
```

Comment:
So is MetaMask the only real prerequisite that's needed and not covered in the guide? Cause I just noticed there's a section on connecting to the testnet and getting funds. If it's something covered in the guide, we shouldn't list it in prerequisites